src/Functions/connectionID.cpp

```diff
@@ -34,7 +34,7 @@ class FunctionConnectionID : public IFunction
 
 void registerFunctionConnectionID(FunctionFactory & factory)
 {
-    factory.registerFunction<FunctionConnectionID>();
+    factory.registerFunction<FunctionConnectionID>(FunctionFactory::CaseInsensitive);
```

Comment:
@alexey-milovidov could you look to https://clickhouse-test-reports.s3.yandex.net/22042/769162765c71ab08f74afcab445a79327673c862/fast_test.html#fail1

I expect CONNECTION_ID should present
should I register all possible aliases in next line?

Reply:
f4d014c